test/rest-api/binary.mocks.ts

```diff
@@ -0,0 +1,9 @@
+import { setupWorker, rest } from 'msw'
+
+const worker = setupWorker(
+  rest.get('https://test.mswjs.io/binary', (_, res, ctx) => {
+    return res(ctx.body(new Uint8Array([1, 2, 3, 4, 5, 6])))
```

Comment:
I wonder, can we technically use an image as a mocked response body here?
That would be a more practical example.

Reply:
Yes, sure but in the end it will just be binary data. I thought the tests are simpler if they don't have to load an image first.

Reply:
I see. I don't mind leaving it here for now and, perhaps, including a more realistic usage example in the docs' recipes.